cg/services/analysis_starter/configurator/file_creators/sample_sheet/raredisease.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import re
+from pathlib import Path
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.apps.lims import LimsAPI
+from cg.constants import SequencingFileTag
+from cg.constants.subject import PlinkPhenotypeStatus, PlinkSex
+from cg.io.gzip import read_gzip_first_line
+from cg.meta.workflow.fastq import _is_undetermined_in_path
+from cg.models.fastq import FastqFileMeta, GetFastqFileMeta
+from cg.models.raredisease.raredisease import (
+    RarediseaseSampleSheetEntry,
+    RarediseaseSampleSheetHeaders,
+)
+from cg.services.analysis_starter.configurator.file_creators.abstract import FileContentCreator
+from cg.services.analysis_starter.configurator.file_creators.utils import get_case_id_from_path
+from cg.store.models import Case, CaseSample, Sample
+from cg.store.store import Store
+
+LOG = logging.getLogger(__name__)
+
+
```

Comment:
This file requires one implementation per pipeline because uses a different base model for sample sheet entries (L46)